src/MovieSnapShot/MovieSnapShot.js

```diff
@@ -56,12 +55,10 @@ class MovieSnapShot extends Component {
             classStyle={"card-container-single-movie"}
             imageStyle={"no-image-hover"}
           />
-          <section className="extra-info-container">
-              <iframe className="extra-info-trailer" src={trailerToPlay} title="Movie Trailer" aria-label={`${this.state.singleMovie.title} trailer`} data-cy="trailer"></iframe>
-            <p className="extra-info" data-cy="overview">{this.state.singleMovie.overview}</p>
-            <p className="extra-info" data-cy="genre">{genres}</p>
-            <p className="extra-info" data-cy="runtime">Runtime: {this.state.singleMovie.runtime} min</p>
-          </section>
+          <MovieInfo 
+            singleMovie={this.state.singleMovie} 
+            trailer={this.state.trailer}
+          />
```

Comment:
Moving this to a component really helped out with the readability of that section of code.